### DISCUSSION TITLE
Welcome to solo Discussions!

### DISCUSSION THREAD

**Discussion by nickray** [solokeys/solo#598] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by goethemarch2** [comment]
I would like to request to add at least the dimensions of somu. The reason is I want to replace the fingerprint sensor in my laptop by SOLO key (and solder it permanently). Would be helpful.
Thanks.

BTW, may be a idea would be create something that is based on the hardware dimensions of popular devices (found in may be thinkpad or other general developer laptops) https://fprint.freedesktop.org/supported-devices.html IIRC, these are USB interface only. (i.e) users can open and replace the fingerprint sensor inside laptop by the U2F key.

https://github.com/solokeys/solo-hw/issues/16

**Comment by kingswitt** [comment]
hello, I came across solo1 from u2f zero, after learning that I can make my own key for cheap with u2f zero software, is that possible with solo1? if so, is there a guide for it?
